src/scripts/docker/docker_prune.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo $CLEAN_VOLUMES
+echo $CLEAN_IMAGES
```

Comment:
If you want to keep these here, maybe make them more explicit:
```suggestion
echo "Clean Volumes: $CLEAN_VOLUMES"
echo "Clean Images: $CLEAN_IMAGES"
```